### PR TITLE
Add UI for a list of rooms

### PR DESCRIPTION
**Pull request by Mariphy**
Added a very simple UI for the list of rooms. There is still a lot of things to work on, but it works.

### CODE REVIEW

**Comment by Mariphy** [comment]
Not sure if I can link it to the issue from here. I'll try to create the next branch from the next issue itself.

File: src/app/(admin)/admin/settings/rooms/page.tsx [hunk 1]
```diff
@@ -6,14 +6,45 @@ import Link from "next/link";
 import { api } from "~/trpc/react";
 
 const Rooms = () => {
+  const { data: rooms, isLoading } = api.rooms.list.useQuery();
   const router = useRouter();
+  console.log(rooms)
```
Comment (dan2811) [inline src/app/(admin)/admin/settings/rooms/page.tsx:11]: Let's remove this log to keep the console clean

File: src/app/(admin)/admin/settings/rooms/page.tsx [hunk 2]
```diff
@@ -6,14 +6,45 @@ import Link from "next/link";
...
+          <thead className="text-left">
+            <tr>
+              <th>Name</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+          <tbody>
+            {rooms?.map(({id, name, description}) => {
+              return (
+                <tr
```
Comment (dan2811) [inline src/app/(admin)/admin/settings/rooms/page.tsx:35]: Add an onclick attribute to the <tr> element that redirects the user to the show page for the room. Something like this should do it:

```suggestion
                <tr
                                  onClick={() => router.push(`/admin/settings/rooms/${id}`)}

```

**Review by dan2811 — CHANGES_REQUESTED** [review]

File: src/app/(admin)/admin/settings/rooms/page.tsx (hunk 1 above)
Comment (Mariphy) [reply src/app/(admin)/admin/settings/rooms/page.tsx]: Done

File: src/app/(admin)/admin/settings/rooms/page.tsx (hunk 2 above)
Comment (Mariphy) [reply src/app/(admin)/admin/settings/rooms/page.tsx:34]: And done

**Comment by Mariphy** [comment]
Could you please merge the branch if all looks fine?

**Pull request merged**